BSPConvertLib/Source/EntityConverter.cs

```diff
@@ -776,6 +776,9 @@ private void GiveAmmoOnStartTouch(Entity trigger, Entity ammoEnt)
 				delay = 0.01f, //hack to make giving ammo happen after setting ammo
 				fireOnce = -1
 			};
+			if (float.Parse(count) < 0)
```

Comment:
This can be simplified slightly by doing the string replace on `ammoOutput` instead.